docs/en/quick-answers/developing/maya-shelf-app-launcher.md

```diff
@@ -26,7 +26,7 @@ try:
     # get the current engine (e.g. tk-maya) 
     current_engine = sgtk.platform.current_engine() 
     if not current_engine: 
-        cmds.error("Shotgun integration is not available!") 
+        cmds.error("ShotGrid integration is not available!") 
```

Comment:
I chose to make this replacement (even though it's in code) as it's an example of an error message being presented to the user.

Reply:
Sounds good.